routes/api.js

```diff
@@ -983,6 +983,19 @@ router.get('/get_users', function(req, res){
   });
 });
 
+router.get('/get_filtered_users', function(req, res){
+  var filter = req.body.filter;
+  c.elasticsearch.search({
+    index: INDEX,
+    type: 'users',
+    body: { query: { match: {'role': 'user'},}},
+    _source: ["id", "firstName", "lastName", "email", "isDeleted", "role"],
+    size: 999 // all
```

Comment:
Does this mean that there can't be more than 999 users? I don't think 999 is all.